### ISSUE TITLE
EChatRoomNotificationLevel.js required twice, causing loading to fail.

### ISSUE THREAD

**Issue by sapphyrus** [DoctorMcKay/node-steam-user#315]
## Describe the bug
It seems like `EChatRoomNotificationLevel.js` is loaded twice in /resources/enums.js, on lines 32 and 36. For some reason the second require call fails (but only on some nodejs versions?), preventing steam-user from being loaded.
```
steam_bot_1  | internal/modules/cjs/loader.js:960
steam_bot_1  |   throw err;
steam_bot_1  |   ^
steam_bot_1  |
steam_bot_1  | Error: Cannot find module '../enums/EChatroomNotificationLevel.js'
steam_bot_1  | Require stack:
steam_bot_1  | - /app/node_modules/steam-user/resources/enums.js
steam_bot_1  | - /app/node_modules/steam-user/index.js
steam_bot_1  | - /app/index.js
steam_bot_1  |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
steam_bot_1  |     at Function.Module._load (internal/modules/cjs/loader.js:840:27)
steam_bot_1  |     at Module.require (internal/modules/cjs/loader.js:1019:19)
steam_bot_1  |     at require (internal/modules/cjs/helpers.js:77:18)
steam_bot_1  |     at Object.<anonymous> (/app/node_modules/steam-user/resources/enums.js:36:40)
steam_bot_1  |     at Module._compile (internal/modules/cjs/loader.js:1133:30)
steam_bot_1  |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
steam_bot_1  |     at Module.load (internal/modules/cjs/loader.js:977:32)
steam_bot_1  |     at Function.Module._load (internal/modules/cjs/loader.js:877:14)
steam_bot_1  |     at Module.require (internal/modules/cjs/loader.js:1019:19) {
steam_bot_1  |   code: 'MODULE_NOT_FOUND',
steam_bot_1  |   requireStack: [
steam_bot_1  |     '/app/node_modules/steam-user/resources/enums.js',
steam_bot_1  |     '/app/node_modules/steam-user/index.js',
steam_bot_1  |     '/app/index.js'
steam_bot_1  |   ]
steam_bot_1  | }
```
Removing the second require call fixes it.
I was, unfortunately, unable to reproduce this issue on my main machine and I have no idea why a second require call would suddenly be unable to find the module.

## Versions
What version of the module and what version of Node.js are you using?
```
root@524dd40a17e4:/app# node -v
v12.16.3
root@524dd40a17e4:/app# npm list steam-user
steam_bot@1.0.0 /app
`-- steam-user@4.16.0
```

**Comment by DoctorMcKay** [comment]
Interesting, they're required as `EChatRoomNotificationLevel` and `EChatroomNotificationLevel` (notices the lowercase R in the second one). I guess that would be why loading is failing only on Unix.

**Comment by DoctorMcKay** [comment]
Fixed in v4.16.1.

**Comment by sapphyrus** [comment]
> Interesting, they're required as `EChatRoomNotificationLevel` and `EChatroomNotificationLevel` (notices the lowercase R in the second one). I guess that would be why loading is failing only on Unix.

Ohh that makes total sense, no idea how I didn't catch that. Thanks for the quick fix
